asllib/Native.ml

```diff
@@ -226,226 +225,141 @@ module NativeBackend = struct
   let bitvector_length bv =
     let bv = as_bitvector bv in
     Bitvector.length bv |> v_of_int
-end
-
-module NativePrimitives = struct
-  open NativeBackend
-
-  let return_one v = return [ return v ]
-
-  let uint = function
-    | [ NV_Literal (L_BitVector bv) ] ->
-        L_Int (Bitvector.to_z_unsigned bv) |> nv_literal |> return_one
-    | [ v ] -> mismatch_type v [ T_Int None ]
-    | li -> Error.fatal_unknown_pos @@ Error.BadArity ("UInt", 1, List.length li)
 
-  let sint = function
-    | [ NV_Literal (L_BitVector bv) ] ->
-        L_Int (Bitvector.to_z_signed bv) |> nv_literal |> return_one
-    | [ v ] -> mismatch_type v [ default_t_bits ]
-    | li -> Error.fatal_unknown_pos @@ Error.BadArity ("SInt", 1, List.length li)
+  module Primitives = struct
+    let return_one v = return [ return v ]
+
+    let uint = function
+      | [ NV_Literal (L_BitVector bv) ] ->
+          L_Int (Bitvector.to_z_unsigned bv) |> nv_literal |> return_one
+      | [ v ] -> mismatch_type v [ integer' ]
+      | li ->
```

Comment:
We have a lot of changes that are due to restructuring of lines that essentially amount to whitespacing.
Perhaps we should apply ocamlformat to all our files?
That would ensure a consistent standard and avoid all those pesky differences.